spec/acceptance/suites/default/10_install_hirs_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'spec_helper_acceptance'
+
+test_name 'hirs_provisioner class'
+
+describe 'hirs_provisioner class' do
+
+  #create a local repo with the necessary HIRS rpms
+  #this can be replaced later when the packages are signed and added to the extras repo
+  def create_local_repo(hirs_host)
+    os = fact_on(hirs_host,'operatingsystemmajrelease')
+    hirs_host.install_package('createrepo')
+    on hirs_host, 'mkdir /usr/local/repo'
+    if os.eql?('7')
+      on hirs_host, 'cd /usr/local/repo; curl -L -O https://github.com/nsacyber/HIRS/releases/download/v1.0.2/HIRS_Provisioner_TPM_2_0-1.0.2-1541093721.d1bdf9.el7.x86_64.rpm'
+      on hirs_host, 'cd /usr/local/repo; curl -L -O https://github.com/nsacyber/HIRS/releases/download/v1.0.2/HIRS_Provisioner_TPM_1_2-1.0.2-1541093721.d1bdf9.el7.noarch.rpm'
+    else os.eql?('6')
+      on hirs_host, 'cd /usr/local/repo; curl -L -O https://github.com/nsacyber/HIRS/releases/download/v1.0.2/HIRS_Provisioner_TPM_1_2-1.0.2-1541093721.d1bdf9.el6.noarch.rpm'
+    end
+    on hirs_host, 'cd /usr/local/repo; curl -L -O https://github.com/nsacyber/HIRS/releases/download/v1.0.2/tpm_module-1.0.2-1541093721.d1bdf9.x86_64.rpm'
+    on hirs_host, 'cd /usr/local/repo; curl -L -O https://github.com/nsacyber/paccor/releases/download/v1.0.6r3/paccor-1.0.6-3.noarch.rpm'
+    on hirs_host, 'createrepo /usr/local/repo'
+    on hirs_host, 'printf "[local.repo]\nname=local\nbaseurl=file:///usr/local/repo\nenabled=1\ngpgcheck=0" > /etc/yum.repos.d/local.repo'
+  end
+
+  #install an aca for the provisioners to talk to
```

Comment:
Should the aca have been set up in the first test?  Trying to understand how the test shows a difference in behavior depending upon the availability of TPMS.

Reply:
The aca is a separate system that the HIRS Provisioner checks in with and uploads and retrieves certificates from.  Since the HIRS Provisioner isn't installed prior to this point there is no need for aca earlier.